lib/dockerize_rails/docker_commands/docker_start.rb

```diff
@@ -0,0 +1,106 @@
+module DockerizeRails
+  module DockerCommands
+    module DockerStart
+      def self.start_rails
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for start_rails is too high. [25.36/20] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Method has too many lines. [20/15] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)

<!-- Comment by [SideCI](https://sideci.com) -->